python/app.py

```diff
@@ -0,0 +1,26 @@
+import time
+import os
+from urllib.parse import urljoin
+
+from flask import Flask
+import jwt
+
+app = Flask(__name__)
+
+METABASE_SITE_URL = os.environ.get('METABASE_SITE_URL', 'http://localhost:3000')
+METABASE_EMBEDDING_SECRET = os.environ.get('METABASE_EMBEDDING_SECRET')
```

Comment:
Let's add a default value here. I ran the sample without reading the readme first. And it failed to run.

```suggestion
METABASE_EMBEDDING_SECRET = os.environ.get('METABASE_EMBEDDING_SECRET', 'ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff')
```

this key is used in most of our example keys, so it should be safe.

Reply:
Does it work with that sample key? If not, it might be better to check and raise an error that the env key needs to be set before running, otherwise it will still be confusing. If we don't need to be fancy, changing get to `os.environ['METABASE_EMBEDDING_SECRET']` would work as it will automatically raise a key error when not yet

Reply:
I think as @maxzheng says, adding a default value is just going to cover up the fact that the secret is missing. I prefer an explicit error.

Reply:
Cool. I'm on board with you guys. My gripe was that the error was quite cryptic. If `os.environ['METABASE_EMBEDDING_SECRET']` throws better error that you need to set that key then let's do that.